tests/roles/backend_services/tasks/main.yaml

```diff
@@ -32,119 +32,38 @@
         oc set data secret/osp-secret "PlacementPassword={{ placement_password }}"
     {% endif %}
 
-- name: deploy backend services
+- name: when not a periodic CI job use the base deployment
+  when: not periodic|default(false)
   ansible.builtin.shell: |
     {{ shell_header }}
     {{ oc_header }}
-    oc apply -f - <<EOF
-    apiVersion: core.openstack.org/v1beta1
-    kind: OpenStackControlPlane
-    metadata:
-      name: openstack
-    spec:
-      secret: osp-secret
-      storageClass: local-storage
-
-      cinder:
-        enabled: false
-        template:
-          cinderAPI: {}
-          cinderScheduler: {}
-          cinderBackup: {}
-          cinderVolumes: {}
-
-      dns:
-        enabled: true
-        template:
-          externalEndpoints:
-          - ipAddressPool: ctlplane
-            loadBalancerIPs:
-            - 192.168.122.80
-          options:
-          - key: server
-            values:
-            - 192.168.122.1
-          replicas: 1
-
-      glance:
-        enabled: false
-        template:
-          glanceAPIInternal: {}
-          glanceAPIExternal: {}
-
-      horizon:
-        enabled: false
-        template: {}
-
-      ironic:
-        enabled: false
-        template:
-          ironicConductors: []
-
-      keystone:
-        enabled: false
-        template: {}
-
-      manila:
-        enabled: false
-        template:
-          manilaAPI: {}
-          manilaScheduler: {}
-          manilaShares: {}
-
-      mariadb:
-        templates:
-          openstack:
-            storageRequest: 500M
-
-      memcached:
-        enabled: true
-        templates:
-          memcached:
-            replicas: 1
-
-      neutron:
-        enabled: false
-        template: {}
-
-      nova:
-        enabled: false
-        template: {}
-
-      ovn:
-        enabled: false
-        template:
-          ovnController:
-            external-ids:
-              system-id: "random"
-              ovn-bridge: "br-int"
-              ovn-encap-type: "geneve"
-
-      placement:
-        enabled: false
-        template: {}
-
-      rabbitmq:
-        templates:
-          rabbitmq:
-            externalEndpoint:
-              loadBalancerIPs:
-              - 172.17.0.85
-              ipAddressPool: internalapi
-              sharedIP: false
-            replicas: 1
-          rabbitmq-cell1:
-            externalEndpoint:
-              loadBalancerIPs:
-              - 172.17.0.86
-              ipAddressPool: internalapi
-              sharedIP: false
-            replicas: 1
-
-      telemetry:
-        enabled: false
-        template: {}
-    EOF
+    mkdir -p tmp/kustomize
+    kustomize build base > tmp/kustomize/test_deployment.yaml
+    oc apply -f tmp/kustomize/test_deployment.yaml
+  args:
+    chdir: "tests/config"
```

Comment:
it didn't like the relative path

https://logserver.rdoproject.org/63/63/6cd31ceb1b30251868779b345cdc466b6fa9718d/github-check/centos-9-standalone-to-crc-data-plane-adoption/c93797f/logs/subnode-1/home/zuul/test_minimal_out_2023-07-12T09:50:36EDT.log.txt.gz

fatal: [localhost]: FAILED! => {"changed": false, "cmd": "set -euxo pipefail\n\n\nmkdir -p tmp/kustomize\nkustomize build base > tmp/kustomize/test_deployment.yaml\noc apply -f tmp/kustomize/test_deployment.yaml\n", "delta": null, "end": null, "msg": "Unable to change directory before execution: [Errno 2] No such file or directory: b'tests/config'", "rc": null, "start": null, "stderr": "", "stderr_lines": [], "stdout": "", "stdout_lines": []}


i'll revisit today see what else we can do

Reply:
@marios The workdir likely already is `<repo>/tests/playbooks` (as i mentioned in https://github.com/openstack-k8s-operators/data-plane-adoption/pull/63#discussion_r1259683932 ).

I think we should try `chdir: ../config` here then? Not sure if it'll work but i think it's worth a try :)

Reply:
I just requested a node hold so i can dig and work it out ... might be we need ./tests/config (the '.') not sure yet

Reply:
and the thing i pointed to is not the final 'call'... we indeed have chdir to the root but that is to call make
it then calls ansible-playbook again... so could well be ../config :)

Reply:
@jistr fyi ../config worked :) so that is right. it doesn't even change to the directory of the currently executing task or role, but instead, it uses the directory of the playbook that invoked the role/tasks.

Reply:
updating